lib/devices/core.js

```diff
@@ -192,7 +192,36 @@ module.exports = function core(device) {
    * @return {void}
    */
   device.getPowerState = function(callback) {
-    command(commands.getPwrState, null, callback);
+    command(commands.getPwrState, null, function(err, data) {
+      if (err) { return callback(err); }
+      var batteryState = "";
+      switch (data.data[1]) {
+        case 0x01:
+          batteryState = "Charging";
+          break;
+      }
+      if (data.data[1] === 0x01) {
+        batteryState = "Charging";
+      } else if (data.data[1] === 0x02) {
+        batteryState = "OK";
+      } else if (data.data[1] === 0x03) {
+        batteryState = "Low";
+      } else if (data.data[1] === 0x04) {
+        batteryState = "Critical";
+      }
+
+      var batteryVoltage = data.data.slice(2, 4).readUInt16BE(0) / 100;
+      var numCharges = parseInt(data.data[4], 16);
+      var obj = {
+        recVer: data.data[0],
+        batteryState: data.data[1],
+        batteryStateS: batteryState,
+        batteryVoltage: batteryVoltage,
+        chargeCount: numCharges,
+        secondsSinceCharge: data.data[4],
+      };
+      callback(null, obj);
+    });
```

Comment:
This is why the second test fails.

@jamuus not sure why you are doing this, doesn't seem related to the PR title

```
  2) Core commands #getPowerState calls #command with params:
     AssertionError: expected command to have been called with arguments 0, 32, null, function spy() {}
    command(0, 32, null, function () {})
      at Context.<anonymous> (spec/lib/devices/core.spec.js:66:34)
```